LIPS/lip-28.md

```diff
@@ -0,0 +1,106 @@
+    
+  Once activated, Tiebreaker can do two things:
+    
+ 1. Execute the blocked proposal in Dual Governance pipeline
+ 2. Unpause any of the paused protocol contracts
+  
+  Tiebreaker is proposed to be set up as “multisig of multisigs”, with 2/3 quorum on top multisig, and independent quorum settings for underlying multisigs.
+
+## Known risks and limitations
+
+The list of the known risks and limitations of the overall design and implementation details can be found here: https://github.com/lidofinance/dual-governance/blob/3e0f1ae5740ef8410e928f6cc106e3a5f45a5a75/docs/known-risks-and-limitations.md
+
+## Notes on deployment
+
+### Roles to be shifted
+
+In order for Dual Governance to be useful, it must have specific controls in the Lido on Ethereum protocol. There’s a number of “control types” used across Lido on Ethereum smart contracts (contract “ownership”, Aragon ACL roles, non-Aragon ACL roles are the main groups). The actual controls and control-over-controls is distributed between a small number of contracts: Aragon Voting, Aragon Agent, EVMScriptExecutor (EasyTrack motions execution gadget) and specific committees (i.e. GateSeal).
+
+The general idea of the transition to Dual Governance is: 1) Aragon Voting to renounce the in-protocol roles / pass role management to Aragon Agent, so that Aragon Voting can’t “do things” directly in the Lido on Ethereum protocol; 2) Aragon Agent to grant the roles for executing on its’ behalf to Dual Governance Executor contract; 3) EasyTrack setup remains as-is (Aragon Voting can change any settings, but can’t grant protocol-specific roles, which are managed by Aragon Agent under Dual Governance); 4) committees setup remains as-is.
+
+### Smoke test plan
+
+There’s a proposal to host a smoke test on Dual Governance smart contracts before proposing the DAO to switch the mechanics on in mainnet. Smoke test is to trial emergency activation and execution flows before the DAO moves the smart contract roles and permissions under the Dual Governance, allowing for contingencies where needed.
+
+1. Deploy the whole setup with temporary `emergency governance` multisig, so that the contracts can be properly set up before the DAO vote;
+2. Trigger the emergency reset through Emergency Activation and Execution committees; 
+3. Set the contracts up as described on the params list through the temporary `emergency governance`, removing the temporary parts altogether;
+4. Start the Lido DAO Aragon vote to set the onchain rights roles;
+5. Submit the “revoke agent forward role from voting” as the Dual Governance action in the vote.
+6. Once executed, the motion fully switches on Dual Governance mechanics for Lido on Ethereum protocol.
+
+## Operational concerns
```

Comment:
5. Vote maturity and enactment periods might be overlapped and shouldn't assume any exact ordering upon their respective launch phases

Reply:
:thinking: what to do on the _known_ compromise of the PK inside the respective msig (sry for the bad timing ask, ugh)